src/components/Nav.vue

```diff
@@ -87,6 +60,24 @@
 
             <hr class="navbar-divider" />
 
+            <template v-if="getLog && (hasUserAccountValidationRights || hasCreditRequestValidationRights)">
+              <router-link
+                v-if="hasUserAccountValidationRights"
+                to="/admin/pending-accounts"
+                class="navbar-item"
+              >
+                Comptes
```

Comment:
I know you didn't change the title, but now, because it isn't anymore below "Admin", "Comptes" might be misleading or not really clear. So It might be a good idea to be more explicit about this entry. "Demandes de création de compte", ou "Comptes en attente de création"... but I suppose this is way to long (is it ?). How about "Créations de compte" ou "Demandes de comptes" ? I'm not very well inspired.

Reply:
Yup. Good catch.

Reply:
Updated